*** spice3f4/spsmp.c	Wed Feb  3 14:20:50 1993
--- spice3f5/spsmp.c	Sun Jan 30 11:53:10 1994
***************
*** 483,492 ****
  int CreateIfMissing;
  {
  MatrixPtr Matrix = (MatrixPtr)eMatrix;
! ElementPtr Element = Matrix->FirstInCol[Col];
  
  /* Begin `SMPfindElt'. */
      ASSERT( IS_SPARSE( Matrix ) );
      Element = spcFindElementInCol(Matrix, &Element, Row, Col, CreateIfMissing);
      return (SMPelement *)Element;
  }
--- 485,497 ----
  int CreateIfMissing;
  {
  MatrixPtr Matrix = (MatrixPtr)eMatrix;
! ElementPtr Element;
  
  /* Begin `SMPfindElt'. */
      ASSERT( IS_SPARSE( Matrix ) );
+     Row = Matrix->ExtToIntRowMap[Row];
+     Col = Matrix->ExtToIntColMap[Col];
+     Element = Matrix->FirstInCol[Col];
      Element = spcFindElementInCol(Matrix, &Element, Row, Col, CreateIfMissing);
      return (SMPelement *)Element;
  }
